File: task2_pangram.cpp
#include <iostream>
#include <string>
using namespace std;
 
int main() {
    string s;
    cout << "Введите строку: ";
    getline(cin, s);
 
    bool seen[26] = {false};
    for (char c : s) {
        if (c >= 'a' && c <= 'z')
            seen[c - 'a'] = true;
    }
 
    bool isPangram = true;
    for (int i = 0; i < 26; i++) {
        if (!seen[i]) { isPangram = false; break; }
    }
 
    cout << (isPangram ? "Yes" : "No") << endl;
    return 0;
}

File: task2_pangram.go
package main
 
import (
	"bufio"
	"fmt"
	"os"
)
 
func main() {
	reader := bufio.NewReader(os.Stdin)
	fmt.Print("Введите строку: ")
	s, _ := reader.ReadString('\n')
 
	seen := [26]bool{}
	for _, c := range s {
		if c >= 'a' && c <= 'z' {
			seen[c-'a'] = true
		}
	}
 
	for _, v := range seen {
		if !v {
			fmt.Println("No")
			return
		}
	}
	fmt.Println("Yes")
}

